editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5667,7 +5668,6 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	animation_hb->add_child(key_auto_insert_button);
 
 	animation_menu = memnew(MenuButton);
-	animation_menu->set_text(TTR("Animation"));
```

Comment:
Can you give it a tooltip instead? (ideally more detailed than the name was, since it's not obvious what an Animation menu would contain).

Reply:
Done!

Reply:
I know @YeldhamDev will want to add a missing dot at the end of the tooltip's sentence, so we might as well do it now :P

Reply:
@akien-mga You know me so well! :wink:

Reply:
I actually copied the style of "Skeleton Options" and "Snapping Options" but forgot to capitalize words :P 

Which one do you prefer?
1. "Animation Key and Pose Options"
2. "Animation key and pose options."

Reply:
Considering that it's phrased like a title, I think the first option is better.

Reply:
Changed to `"Animation Key and Pose Options"`.